docs/src/en/guide/get-started/README.md

```diff
@@ -79,14 +79,17 @@ Welcome to Waline. In just a few steps, you can enable Waline to provide comment
 
 ## Importing in HTML (Client)
 
-Make the following settings on your web page:
+Here is how you can implement Waline on your web page:
```

Comment:
```diff
- Here is how you can implement Waline on your web page:
+ Here is how you can import Waline on your web page:
```

Reply:
I am not sure, but I do think `implement` is not better, that sound like achieve something, but waline is just tool.

Anyway, these improvements are nice, and we welcome you to improve other pages if you have time.

Reply:
You can use "add", "embed", "import", etc. Whatever makes the most sense to you.